### PR TITLE
[clipboard] add share button to clipboard url

### LINKED ISSUE #28: Add button to share URL
Add the capability to share a specific post with a button on the card.

### PR DESCRIPTION
**Pull request by JuanequeX**
## Quick Info
- Add clipboard.js to enable a button that automatically copies the relative path of each post to the clipboard, allowing users to easily share this specific TIL post with the community.

## Screenshots (if apply)

https://github.com/OswaldoPineda/Today_I_learned/assets/50384228/f99ae51e-4246-4541-b39c-2be593845c1f

### CODE REVIEW

File: app/javascript/clipboard_config.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import ClipboardJS from 'clipboard';
```
Comment (OswaldoPineda) [inline app/javascript/clipboard_config.js:1]: change var variables to const

**Review by OswaldoPineda — CHANGES_REQUESTED** [review]

File: app/javascript/clipboard_config.js (hunk 1 above)
Comment (JuanequeX) [reply app/javascript/clipboard_config.js:1]: DONE

**Review by OswaldoPineda — DISMISSED** [review]

**Comment by JuanequeX** [comment]
please give me an overview to unblock this PR, because this branch was pushed to production currently. @OswaldoPineda

**Review by OswaldoPineda — APPROVED** [review]
LGTM!

**Pull request merged**